scripts.js

```diff
@@ -1,77 +1,263 @@
-var i = 1;
+/*declaracion de variables:
+<------------------------------------------------------------------>*/
 
-class Ludo {
-	
-	constructor() {
-		this.tirarDado = this.tirarDado.bind(this)
-		this.botonJugar = document.getElementById('BotonInicio')
-		this.dado = document.getElementById('dado')
-		this.posDado = 1  //numero de referencia para la animacion del dado que varia entre 1 y 0
-		this.nCasillas = 52
-		this.obtenerCasillas()
-		this.ocultarBoton()
-		this.habilitarDado()
+//Arreglos que contienen los elementos HTML que conforman las casillas 
+const casilla = 
+[{}, {}, {}, {}, {}, {}, 
+{}, {}, {}, {}, {}, {}, 
+{}, {}, {}, {}, {}, {},
+{}, {}, {}, {}, {}, {},	//Arreglo que contiene los elementos HTML
+{}, {}, {}, {}, {}, {},	//que conforman las casillas blancas del trablero
+{}, {}, {}, {}, {}, {},
+{}, {}, {}, {}, {}, {},
+{}, {}, {}, {}, {}, {},
+{}, {}, {}, {}]
```

Comment:
```suggestion
const casilla = Array(52).fill({})
```
Oh no recuerda, siempre que la tarea es repetitiva seguro hay una mejor forma de hacerla

Reply:
oh, no. ya había probado hacerlo así antes, y no funciono como lo esperaba:
Por lo que pude apreciar, al llenar el arreglo de esa manera con un objeto, se llena con algo parecido a apuntadores que apuntan a un mismo objeto. Entonces, cualquier cambio que se la haga a cualquier elemento del arreglo, se hace en cada elemento del arreglo (lo que sospecho que son apuntadores al mismo objeto, y son, de hecho, cambios a un solo objeto). Por eso lo he terminado haciendo de esa forma tan engorrosa